.codepipeline/buildspec-tests-build.yml

```diff
@@ -52,13 +46,13 @@ phases:
   build:
     on-failure: CONTINUE
     commands:
-      - make test-broadcast-flow; exit 0
+      # - make test-broadcast-flow; exit 0
```

Comment:
Why comment out the code, if it's being left for a reason add a comment to say why it's commented out?